src/functionalTest/resources/scenarios/RIA-1976-hearing-notice-document-iac-birmingham.json

```diff
@@ -1,10 +1,10 @@
 {
-  "description": "RIA-1976 Create hearing notice PDF (IAC Birmingham)",
+  "description": "RIA-2970 Create hearing adjourned without date notice PDF",
   "request": {
     "uri": "/asylum/ccdAboutToSubmit",
     "credentials": "AdminOfficer",
     "input": {
-      "eventId": "listCase",
+      "eventId": "adjournHearingWithoutDate",
       "state": "listing",
       "caseData": {
         "template": "minimal-appeal-submitted.json",
```

Comment:
Is this function test no longer needed? Or is this a mistake forgetting to create a new .json whilst editing?
The filename does not look like its for RIA-2970 ticket.

Reply:
hey, yeah, good catch! I totally missed that. I change it now...